src/node_sea.cc

```diff
@@ -75,9 +77,12 @@ size_t SeaSerializer::Write(const SeaResource& sea) {
   DCHECK_EQ(written_total, SeaResource::kHeaderSize);
 
   Debug("Write SEA resource code %p, size=%zu\n",
```

Comment:
nit: When `useSnapshot` is enabled, this would be writing the snapshot instead of the code. What do you think about printing `snapshot` instead of `resource code` in that case?